cassandra/cqlengine/functions.py

```diff
@@ -12,20 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import division
 from datetime import datetime
 
 from cassandra.cqlengine import UnicodeMixin, ValidationError
 
 import sys
 
-if sys.version_info >= (2, 7):
-    def get_total_seconds(td):
-        return td.total_seconds()
-else:
-    def get_total_seconds(td):
-        # integer division used here to emulate built-in total_seconds
-        return ((86400 * td.days + td.seconds) * 10 ** 6 + td.microseconds) / 10 ** 6
+def get_total_seconds(td):
+    return td.total_seconds()
```

Comment:
Hey @bschoening, this one is also removed by [another PR](https://github.com/datastax/python-driver/pull/1201) that I'm looking to merge in order to clean up the mock usages.  Would you mind rebasing this against master once I get that PR in?  Should go in later today.

Reply:
To follow up: the other PR has been merged.  Should be a pretty straightforward conflict resolution @bschoening ... I'm really hoping I didn't inadvertently mess you up here. :(

Reply:
@absurdfarce no problem, I'll rebase it.